test/testcafe/framework/page-objects/ChallengeCustomappPushPageObject.js

```diff
@@ -0,0 +1,44 @@
+import ChallengeFactorPageObject from './ChallengeFactorPageObject';
+
```

Comment:
nit: name of this file can have `App` camelCase, so
`ChallengeCustomappPushPageObject.js` -> `ChallengeCustomAppPushPageObject.js`